### PR TITLE
Add an example of prometheus settings

### PR DESCRIPTION
**Pull request by lenkis**

### CODE REVIEW

File: metrics/plugins/prometheus/README.md [hunk 1]
```diff
@@ -26,3 +26,18 @@ local httpd = require('http.server').new(...)
 ...
 httpd:route( { path = '/metrics' }, prometheus.collect_http)
```
Comment (olegrok) [inline metrics/plugins/prometheus/README.md:27]: Here is an example of http.server v1, in your patch you use http v2. I think both parts should use one version.

File: metrics/plugins/prometheus/README.md [hunk 2]
```diff
@@ -26,3 +26,18 @@ local httpd = require('http.server').new(...)
...
+
+### Settings
+
+An example:
+```lua
+metrics = require('metrics')
+metrics.enable_default_metrics()
+
+prometheus = require('metrics.plugins.prometheus')
+metrics = require('http.server').new('0.0.0.0', 8080)
```
Comment (olegrok) [inline metrics/plugins/prometheus/README.md:38]: It's better to use "httpd" or "http_server" here. It clashes with the first line.

Comment (olegrok) [reply metrics/plugins/prometheus/README.md:38]: I mean
```lua
httpd = require('http.server').new('0.0.0.0', 8080)
router = require('http.router').new({charset = "utf8"})
httpd:set_router(router)
router:route( { path = '/metrics' }, prometheus.collect_http)
httpd:start()
```

File: metrics/plugins/prometheus/README.md (hunk 1 above)
Comment (olegrok) [reply metrics/plugins/prometheus/README.md:27]:
```lua
local httpd = require('http.server').new(...)
local router = require('http.router').new(...)
httpd:set_router(router)
...
router:route( { path = '/metrics' }, prometheus.collect_http)
```

**Comment by vasiliy-t** [comment]
would be nice if you could add changelog entry describing this update

**Pull request merged**